src/plugins/camera/camera_impl.cpp

```diff
@@ -1633,14 +1633,15 @@ void CameraImpl::refresh_params()
         // any other possible settings to change. However, we still would
         // like to notify the current settings with this one change.
         notify_current_settings();
+        notify_possible_setting_options();
```

Comment:
This one line is the actual fix I needed :sweat_smile:

Reply:
Haha, ok!